api/src/main/java/com/codeforcommunity/dto/emailer/EmailAttachment.java

```diff
@@ -0,0 +1,111 @@
+package com.codeforcommunity.dto.emailer;
+
+import com.codeforcommunity.aws.EncodedImage;
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.BadRequestImageException;
+import com.codeforcommunity.exceptions.HandledException;
+
+import org.simplejavamail.api.email.AttachmentResource;
+
+import javax.mail.util.ByteArrayDataSource;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+
+public class EmailAttachment extends ApiDto {
+    private String name;
+    private String data;
+
+    public EmailAttachment(String name, String data) {
+        this.name = name;
+        this.data = data;
+    }
+
+    private EmailAttachment() {}
+
+    private static String getFileExtension(String base64Image) {
```

Comment:
Adding onto that, since this method is getting a file's mime type, could we rename it so it better describe what it's doing - maybe `getFileMimeType`?